### PR TITLE
feat(ContractEditor): connect editor - 23

### PR DESCRIPTION
**Pull request by DianaLease**
https://github.com/accordproject/template-studio-v2/issues/23

feat(templates): add load template object action/saga/reducer and pass to contract editor
feat(clause): add parse clause function and pass to contract editor
feat(clause): add clause actions and reducer to populate store with parse result or error
chore(webpack): update webpack config for css import in cicero-ui

*FLAGS:*
- Currently, this uses a utility function and not a saga for parsing because we need the function to return the result in addition to updating the store. Once we figure out how and where we are using this result/error, we can probably move the utility function into a saga and have the contract editor in cicero-ui listen to the "clauseState" part of the store instead of receiving the result as returned from the utility function.
- Corresponds with https://github.com/accordproject/cicero-ui/pull/39

### CODE REVIEW

**Review by jeromesimeon — DISMISSED** [review]
LGTM!

File: src/sagas/templatesSaga.js [hunk 1]
```diff
@@ -33,7 +36,21 @@ export function* addNewTemplateToStore() {
   yield put(actions.addNewTemplateSuccess(newTemplate));
 }
 
+/**
+ * saga which checks if template is in the store
+ * and loads the template if it is not
+ */
+export function* addTemplateObjectToStore(action) {
```
Comment (jolanglinais) [inline src/sagas/templatesSaga.js:43]: This could have a try/catch and thus an error action associated with it.

Comment (DianaLease) [reply src/sagas/templatesSaga.js:43]: good call. will add, thanks!

Comment (jolanglinais) [reply src/sagas/templatesSaga.js:43]: Would this be integrated with the `Error` container in the future?

Comment (DianaLease) [reply src/sagas/templatesSaga.js:43]: I'm not sure if it makes sense to surface this type of error in the error logger (which is more for errors from a user editing a template to help them edit it correctly). I think the UI will dictate where these type of errors should be (maybe in a separate message-like component?).

**Review by jolanglinais — DISMISSED** [review]
Great!

**Review by jolanglinais — APPROVED** [review]

**Pull request merged**